client/tests/integration.test.js

```diff
@@ -17,10 +18,14 @@ describe('login actions dispatchers', () => {
             login: {
                 isLoggedIn: true,
                 username: 'foobar',
-                name: 'Foo Bar'
+                name: 'Foo Bar',
+                email: 'foo@bar.com'
             },
             cycles: {
                 cycles: []
+            },
+            ats: {
+                ats: []
```

Comment:
For my recommendation above, these should be changed to: 

```js
cycles: [], 
ats: [],
```

Reply:
Seeing these comments makes me think we should handle the actions/reducers a bit differently. @rwaldron I added more information to the "cycles" reducer here in a way that we can't simplify the cycles state to just `[]`: https://github.com/bocoup/aria-at-report/pull/34/files#diff-3b3898c55e860045e6333773efb63da8

I think having a whole `action` and `reducer` for each item we want to keep in global state is a little bit extreme.. which is the direction Rick's comments push us in. On the other hand, I think it's going to get confusing what item should be in which `reducer` file (which is then imported in `reducers/index.js` and will lead to things like `state.cycle.cycle`. Right now -- I'm thinking perhaps we should put all the actions in one file and all the reducers in one file. Or, we should really architect how they should be split up -- because right now I'm not positive how to split them up.

Reply:
> On the other hand, I think it's going to get confusing what item should be in which reducer file (which is then imported in reducers/index.js and will lead to things like state.cycle.cycle

That's where you're model is now: `state.cycles.cycles`. If revise and follow my recommendation, your `cycles` reducer will only deal with reducing for `cycles`, then it can always just return that `cycles` array. You don't even need to split things into different files, you just need to reorganize them structurally and not rely on default exports. I will comment on your changeset with how I imagine that can be done.

Reply:
@spectranaut https://github.com/bocoup/aria-at-report/pull/34#discussion_r408991800